comp/core/workloadmeta/collectors/internal/nvml/nvml.go

```diff
@@ -90,6 +90,8 @@ func (c *collector) getGPUdeviceInfo(device nvml.Device) (*workloadmeta.GPU, err
 	if ret != nvml.SUCCESS {
 		return nil, fmt.Errorf("failed to get GPU index ID: %v", nvml.ErrorString(ret))
 	}
+	//we try to get the driver version as a best effort, don't care if we fail
+	driver, _ := c.nvmlLib.SystemGetDriverVersion()
```

Comment:
how often it can fail? I'm wondering if we should still return a debug log to ease investigation.